src/meta/src/stream/stream_graph/actor.rs

```diff
@@ -714,7 +714,9 @@ impl ActorGraphBuilder {
             .values()
             .map(|d| d.parallelism())
             .sum::<usize>() as u64;
-        let id_gen = GlobalActorIdGen::new(&id_gen_manager, actor_len).await?;
+
+        // TODO: use sql_id_gen that is not implemented yet.
+        let id_gen = GlobalActorIdGen::new(env.id_gen_manager(), actor_len).await?;
```

Comment:
Will replace the id_gen of sql-backend version in meta env in later PR, so we changed the parameter of id_gen to meta env.